pkg/templates/aws/edges/cloudfront_distribution-load_balancer.yaml

```diff
@@ -0,0 +1,88 @@
+source: aws:cloudfront_distribution
+target: aws:load_balancer
+always_process: true
+operational_rules:
+  - configuration_rules:
+      - resource: '{{ .Source }}'
+        configuration:
+          field: Origins
+          value:
+            - CustomOriginConfig:
+                HttpPort: 80
+                HttpsPort: 443
+                OriginProtocolPolicy: http-only
+                OriginSslProtocols:
+                  - TLSv1.2
+                  - TLSv1
+                  - SSLv3
+                  - TLSv1.1
+              DomainName: '{{ .Target }}#DnsName'
+              OriginId: '{{ .Target.Name }}'
+      - resource: '{{ .Source }}'
+        configuration:
+          field: CacheBehaviors
+          value: |
+            [   
+            {{ $targetOriginId := .Target.Name }}
+            {{ $routes := makeSlice }}
+            {{ $rules := allDownstream "aws:load_balancer_listener_rule" .Target }}
+            {{ range $i, $rule := $rules }}
+              {{ range $j, $condition := (fieldValue "Conditions" $rule) }}
+                {{range $k, $pathPattern := $condition.PathPattern.Values }}
+                  {{- if not (sliceContains $routes $pathPattern) }}
+                    {{ $routes = appendSlice $routes $pathPattern }}
+                  {{- end }}
+                {{- end }}
+              {{ end }}
+            {{ end }}
```

Comment:
This kind of stuff is exactly why I wish we could use an actual scripting language instead of making our own. No action item, just commentary.

Reply:
yeah i think we can find ways to simplify this